.github/workflows/dotnet-build.yml

```diff
@@ -40,6 +40,16 @@ jobs:
         fetch-depth: 1000
         fetch-tags: true
 
+    #--------------INIT--------------
+
+    - name: PR title validator
```

Comment:
We need the version tag prefix to be available for this script, so that it can generate the new version labels for pull request builds. And dotnet and npm currently have different prefixes so we cannot keep this step in the common workflow. This is what was causing the pr title validator script to fail until now, even though the error was silenced.

Reply:
Well that is the problem, this is not really a common job, since it creates language specific labels to be used during a PR run. In the event of both npm and dotnet files being changed, the PR title validator will not work without making a lot of changes to it and following steps.

Reply:
The purpose of these labels is to be able to always generate version numbers even during Pull Request runs, so that we can use them if we need to create deployments from a PR. But after going over the pipelines, we are currently not utilising this, so we can drop this functionality and we can make it a common job again, what do you think?

Reply:
I will drop that label generation for now since it is not used anywhere and will make the pr title validator a common job again. The less unused code the better.

Reply:
Updated with latest commit and tested in [this draft PR](https://github.com/WMS-Ceataec/standardised-vessel-dataset/pull/25)